automation/browser.py

```
@@ -15,7 +16,7 @@ class BrowserManager:
     def __init__(self, browser_type: str = "msedge", headless: bool = False):
         """
         初始化浏览器管理器
-        :param browser_type: 浏览器类型 (chrome, msedge, firefox)
+        :param browser_type: 浏览器类型 (chrome, msedge）
```

Ruff (0.15.5)

[warning] Docstring contains ambiguous `）` (FULLWIDTH RIGHT PARENTHESIS). Did you mean `)` (RIGHT PARENTHESIS)?

(RUF002)